crates/ruff_python_parser/src/typing.rs

```diff
@@ -24,23 +24,30 @@ pub enum AnnotationKind {
 
 /// Parse a type annotation from a string.
 pub fn parse_type_annotation(
-    value: &str,
+    // Parsed contents of the string literal node that represents this type annotation.
```

Comment:
The inline comments are uncommon for Rust. 

It's more common to refer to the arguments using their name.

Parses the value of a string literal `parsed_contents` with `range` as a type annotation.